cmd/snap/cmd_snap_op.go

```diff
@@ -1396,6 +1397,8 @@ func init() {
 			"revision": i18n.G("Remove only the given revision"),
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"purge": i18n.G("Remove the snap without saving a snapshot of its data"),
+			// TRANSLATORS: This should not start with a lowercase letter.
+			"terminate": i18n.G("Kill running snap apps"),
```

Comment:
Perhaps something less voilent: 
```suggestion
			"terminate": i18n.G("Terminate running application processes"),
```

Reply:
Updated, Thanks!